interfaces/builtin/joystick_test.go

```diff
@@ -99,6 +99,7 @@ KERNEL=="event[0-9]*", SUBSYSTEM=="input", ENV{ID_INPUT_JOYSTICK}=="1", TAG+="sn
 	c.Assert(spec.Snippets(), testutil.Contains, `# joystick
 KERNEL=="full", SUBSYSTEM=="mem", TAG+="snap_consumer_app"`)
 	c.Assert(spec.Snippets(), testutil.Contains, `TAG=="snap_consumer_app", RUN+="/usr/lib/snapd/snap-device-helper $env{ACTION} snap_consumer_app $devpath $major:$minor"`)
+	c.Assert(spec.GetTriggeredSubsystem(), Equals, `input/joystick`)
```

Comment:
Should it be possible for a single specification to trigger multiple subsystems? Is that something that udev supports?

Reply:
The manpage says it's possible :)